scripts/license/add.py

```diff
@@ -16,22 +16,29 @@
 Adds a license header to files.
 """
 
+import os
 from pathlib import Path
 import subprocess
 
 from license_header import license_header, has_license_header
 
-PROJECT_ROOT = Path(subprocess.check_output(
-    "git rev-parse --show-toplevel".split()).strip().decode("utf-8"))
+PROJECT_ROOT = subprocess.check_output(
+    "git rev-parse --show-toplevel".split()).strip().decode("utf-8")
+target_dirs = [os.path.join(PROJECT_ROOT, d) for d in os.listdir(PROJECT_ROOT)
```

Comment:
`target_dirs` is used only in `main()`. Please consider moving `target_dirs` into `main()` to minimize the scope.